content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -215,7 +220,7 @@ Here is an example of a correct configuration:
 --kube-reserved=cpu=4,memory=4Gi
 --system-reserved=cpu=1,memory=1Gi
 --memory-manager-policy=Static
---reserved-memory 0:memory=3Gi --reserved-memory 1:memory=2148Mi
+--reserved-memory 0:memory=3Gi;1:memory=2148Mi
```

Comment:
```
--reserved-memory '0:memory=3Gi;1:memory=2148Mi'
```
would be better right? since on linux `;` is a shell command separator.

Reply:
yes. I will make the change.